index.bs

```diff
@@ -3069,14 +3071,25 @@ The [=remote end steps=] with |command parameters| are:
 
 1. Let |viewport parameter| be the |command parameters|["<code>viewport</code>"] field.
 
+1. Let |device pixel ratio parameter| be the |command parameters|["<code>devicePixelRatio</code>"] field.
+
+1. If |device pixel ratio parameter| is not null and the implementation is not able to change the size of the [=CSS Pixel=] in |context|'s [=layout viewport=] for any reason, return [=error=] with [=error code=] [=unsupported operation=].
+
 1. If |viewport parameter| is not null:
     1. Let |width| be the |viewport parameter|["<code>width</code>"]
     1. Let |height| be the |viewport parameter|["<code>height</code>"]
     1. If the |width| or |height| is outside of the implementation-specific or runtime constraints, return an [=error=] with the [=error code=] [=unsupported operation=].
     1. Set the [=layout viewport=] of the |context| to be |width| CSS pixels wide and |height| CSS pixels high.
 1. Else:
     1. Set the [=layout viewport=] of the |context| to the implementation-defined default value.
+
+1. If |device pixel ratio parameter| is not null:
+  1. Change the size of the [=CSS Pixel=] in |context|'s [=layout viewport=] such that it corresponds to |device pixel ratio parameter| device pixels.
+1. Else:
```

Comment:
We generally use "otherwise" rather than "else"

Reply:
Updated